js/build.js

```diff
@@ -2695,8 +2703,19 @@ Fliplet.Widget.instance('chat', function (data) {
 
           Fliplet.UI.Toast.dismiss();
         }
-        $wrapper.removeClass('loading');
-        $wrapper.removeClass('error');
+      }).catch(function (err) {
+        // Log in again if the token does not seem valid
+        $wrapper.addClass('loading');
```

Comment:
What happens if an error occurs but the status code isn't `403`?

It looks like it might be stuck in `loading` mode without any further actions.

Reply:
@tonytlwu made some changes so that existing failures are treated as they were before (with the error message)